casper/src/main/resources/PoS.rho

```diff
@@ -1,180 +1,165 @@
- // ----+------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------
-
-new PoS, rs(`rho:registry:insertSigned:ed25519`), uriOut in {
-
- for (@minimumBond, @maximumBond <- PoS) {
-
-    match "11112GofKy1v5mPWZjZEYBByvpZ4ahYJp6vFN1hgGCn7L543rFSpyA" {
-
-      posRevAddress => {
-
-       new eitherCh,
-          revVaultCh,
-          registryLookup(`rho:registry:lookup`) in {
-
-          registryLookup!(`rho:id:j6trahbxycumerwpr5qype7j43b7eqh8auebwsa9nn68if47gswh73`, *eitherCh)
-          | registryLookup!(`rho:id:1o93uitkrjfubh43jt19owanuezhntag5wh74c6ur5feuotpi73q8z`, *revVaultCh)
-          | for (@(_, Either) <- eitherCh; @(_, RevVault) <- revVaultCh) {
-
-            @RevVault!("findOrCreate", posRevAddress, Nil)
-
-            | new bondsCh,
-              getParameters(`rho:deploy:params`),
-              revAddressOps(`rho:rev:address`),
-              paymentStore,
-              deposit in {
-
-              bondsCh!({})
-
-              | contract PoS (@"getBonds", return) = {
-                for(@bonds <- bondsCh) {
-                  bondsCh!(bonds) | return!(bonds)
-                }
+/*
+The table below describes the required computations and their dependencies
+
+No. | Dependency | Computation method | Result
+----+------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------
+1.  |            | given              | sk = d70491c21a5ba44ffb60a97d9bfbf3f3839b997bde107a3a8ab6872594282740
+2.  |            | given              | timestamp = 1553862086627
+3.  |            | lastNonce          | nonce = 9223372036854775807
+4.  | 1,         | ed25519            | pk = cc87bf7747a8c176714b417ca14a63897b07572876c5e38a7896b6007738ef81
+5.  | 4, 2,      | genIds             | uname = Unforgeable(0xf974f799067b34be1acb7b565e0b160edb23df3e3ad58f200af183ae48b458cd)
+6.  | 3, 5,      | registry           | value = (9223372036854775807, bundle+ {   Unforgeable(0xf974f799067b34be1acb7b565e0b160edb23df3e3ad58f200af183ae48b458cd) })
+7.  | 6,         | protobuf           | toSign = 2a3eaa013b0a0d2a0b10feffffffffffffffff010a2a5a280a243a220a20f974f799067b34be1acb7b565e0b160edb23df3e3ad58f200af183ae48b458cd1001
+8.  | 7, 1,      | ed25519            | sig = 7ff160accda7f76f14f2ae132aaae140ac22b2671e561a58a566bff1a01e49cf372201a5df678aba5ff09171b6af966906deffbf8933efeeddb69a4c204b0503
+9.  | 4,         | registry           | uri = rho:id:cnec3pa8prp4out3yc8facon6grm3xbsotpd4ckjfx8ghuw77xadzt
+----+------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------
+*/
+
+new MakePoS,
```

Comment:
Why did you name this contract `MakePoS` when there is already a `MakePoS` contract? Also, you use `PoS` in the body of the contract, so why not just keep it as it was originally?

Reply:
First of all this is just a local name so it doesn't affect anybody else except the reader of this file.

I named the top-level contract like this because it "makes a PoS contract". The returned value is the actual "PoS".  For me it reads better but I don't have a strong opinion about it.